src/main/java/de/craftlancer/clcapture/CapturePoint.java

```diff
@@ -68,50 +63,47 @@
     private long winTime = -1;
     
     private UUID currentOwner = null;
+    private UUID previousOwner = null;
+    private UUID previousMessageOwner = null;
     private Map<UUID, Integer> timeMap = new HashMap<>();
     private KeyedBossBar bar;
+    //A list of all clans/players that are in the region
     
-    public CapturePoint(CLCapture plugin, String name, String id, CapturePointType type, Block chestLocation, Block signLocation) {
+    public CapturePoint(CLCapture plugin, String name, String id, CapturePointType type, Block chestLocation) {
         Bukkit.getPluginManager().registerEvents(this, plugin);
         this.plugin = plugin;
         this.name = name;
         this.id = id;
-        
         this.type = type;
-        
         this.chestLocation = chestLocation.getLocation();
-        this.signLocation = signLocation.getLocation();
-        
-        updateSign();
+        setRegion(this.chestLocation);
     }
     
     public CapturePoint(CLCapture plugin, String id, ConfigurationSection config) {
         Bukkit.getPluginManager().registerEvents(this, plugin);
         this.plugin = plugin;
         this.id = id;
         this.name = config.getString("name", id);
-        
         this.type = plugin.getPointType(config.getString("type"));
         this.chestLocation = config.getObject("chest", Location.class);
-        this.signLocation = config.getObject("sign", Location.class);
-        
-        updateSign();
+        setRegion(this.chestLocation);
     }
     
-    public Inventory getInventory() {
-        return ((Chest) chestLocation.getBlock().getState()).getBlockInventory();
+    private void setRegion(Location chestLocation) {

```

Comment:
Maybe change to update region and access this.chestLocation directly instead of passing it as parameter. 
You may also want to use org.bukkit.util.BoundingBox instead, it could make the region checking easier/more efficient.